django/forms/widgets.py

```diff
@@ -428,6 +431,9 @@ def value_from_datadict(self, data, files, name):
             return False
         return upload
 
+    def use_required_attribute(self, initial):
```

Comment:
I'm still conflicted. I suppose we can keep the currently implementation for now.